vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/model/ModuleWithSpace.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.diging.vspace.core.services.impl.model;
+import edu.asu.diging.vspace.core.model.impl.Module;
+
+public class ModuleWithSpace extends Module {
```

Comment:
is this any different from the one used on the staff search?

Reply:
It seems it is only used in public search scenario where space info is added to module.